src/package/polyfill/createHookClass.ts

```diff
@@ -2,18 +2,50 @@ import { getMostRecentImpl } from './hmrCache';
 
 export function createHookClass(elementName: string, originalImpl: any) {
     return class extends originalImpl {
-        static get observedAttributes() {
-            return super.observedAttributes;
-        }
-
         connectedCallback() {
             const mostRecentImpl = getMostRecentImpl(elementName).prototype;
+
+            const ctx = this;
+            const observerOptions = {
+                childList: false,
+                attributes: true,
+                subtree: false
+            };
+
+            function callback(mutationList: any[], observer: MutationObserver) {
+                mutationList.forEach(mutation => {
+                    const mostRecentImpl = getMostRecentImpl(elementName).prototype;
+                    const attributes = getMostRecentImpl(elementName).__observedAttributes;
```

Comment:
Can we store `getMostRecentImpl(elementName)` in a variable to not call it twice?

Reply:
Yes, sloppy work of me 😁

Reply:
fixed